src/reconstruct_loopset.jl

```diff
@@ -417,8 +417,26 @@ function _avx_loopset(OPSsv, ARFsv, AMsv, LPSYMsv, LBsv, vargs)
         AMsv, LPSYMsv, LBsv, vargs
     )
 end
+const _body_ = Ref{Any}(nothing)
```

Comment:
This seems clever. Will make debugging easier.

Reply:
Oops, I didn't actually mean to commit that, but if you like it...